tests/installation/installation_overview.pm

```diff
@@ -65,10 +65,9 @@ sub check_default_target {
         get_var('REMOTE_CONTROLLER') || (get_var('BACKEND', '') =~ /spvm|pvm_hmc|ipmi/));
     # exclude non-desktop environment and scenarios with edition of package selection (bsc#1167736)
     return if (!get_var('DESKTOP') || get_var('PATTERNS'));
-    return if (get_var 'BSC1167736');
 
     # Set expectations
-    my $expected_target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
+    my $expected_target = check_var('DESKTOP', 'textmode*') ? "multi-user" : "graphical";
```

Comment:
How is the `*` supposed to work? I don't understand what you want to achieve with that. AFAIK there is no globbing or regex support in `check_var` so you would compare against a literal string `textmode*`. I hope nobody used that in test settings. Maybe you want

```suggestion
    my $expected_target = get_var('DESKTOP', '') =~ qr/textmode/ ? "multi-user" : "graphical";
```

Reply:
textmode and textmode-server should be used in this case. That is a regex, that both are applied.

Reply:
Right. My proposal would work for that. Does `check_var('DESKTOP', 'textmode*')` actually work?

Reply:
Watch my verification run https://openqa.opensuse.org/t2150725 . That is textmode-server, which has been failing before.

Reply:
ok, let's try